dlhub_sdk/client.py

```diff
@@ -125,13 +136,20 @@ def __init__(self, dlh_authorizer: Optional[GlobusAuthorizer] = None,
         }
 
         login_manager = FuncXLoginManager(authorizers=auth_dict)
-        self._fx_client = FuncXClient(login_manager=login_manager)
+        self._fx_client = FuncXClient(funcx_service_address="https://api.dev.funcx.org/v2", login_manager=login_manager)
```

Comment:
We should be deploying to prod at 2:30pm cst today. It might be worth checking this after the deployment and then switching back to prod.

Reply:
@ericblau I see this is still pointing to dev?